### PR TITLE
Refactor env variables and upgrade rc6

### PR DESCRIPTION
**Pull request by danduk82**

### CODE REVIEW

File: templates/generic/service_and_deployment.yaml [hunk 1]
```diff
@@ -83,79 +85,20 @@ spec:
...
-              valueFrom:
-                secretKeyRef:
-                  name: geoserver-rabbitmq
-                  key: rabbitmq-password
-            - name: RABBITMQ_HOST
-              value: {{ $geoserverName }}-rabbitmq
-{{- if $values.geoserver.config.env }}
-{{ toYaml $values.geoserver.config.env | indent 12 }}
-{{- end }}
+          {{ include "geoserver.common.env.variables" $fullEnv | nindent 12}}
```
Comment (sbrunner) [inline templates/generic/service_and_deployment.yaml:90]:
```suggestion
          {{ include "geoserver.common.env.variables" $ | nindent 12}}
```
Should work without any additional variable :-)

File: templates/_helpers.tpl [hunk 2]
```diff
@@ -73,3 +73,67 @@ port: 5672
 username: {{ .Values.rabbitmq.auth.username }}
 password: ${RABBITMQ_PASSWORD}
 {{- end }}
+
+
+{{- define "geoserver.common.env.variables" -}}
+- name: EUREKA_SERVER_URL
+  value: http://{{ include "geoserver.fullname" . }}-discovery:8761/eureka
+- name: BACKEND_CATALOG
+  value: "false"
```
Comment (sbrunner) [inline templates/_helpers.tpl:82]: I prefer to put the static env in the values.yaml to be able to override them :-)

**Review by sbrunner — COMMENTED** [review]
Just come little tips, for the other I'm confident to you :-)

**Review by sbrunner — APPROVED** [review]

File: templates/generic/service_and_deployment.yaml (hunk 1 above)
Comment (danduk82) [reply templates/generic/service_and_deployment.yaml:90]: correct, thanks! I cleanup this

**Pull request merged**